packages/core/src/PartialExecutionUtils/getSourceDataGroups.ts

```diff
@@ -13,6 +13,18 @@ function sortByInputIndexThenByName(
 	}
 }
 
+type Group = {
```

Comment:
What group is this? Could we be more explicit? Is it for example a `SourceConnectionGroup` or `IncomingConnectionGroup`?

Reply:
I renamed it ❤️